README.md

```diff
@@ -172,6 +172,141 @@ which should print out some information about how to use the aligner.
 
 The instructions are the same as for Mac users, with the exception that there is no direct analogue to XCode. Instead, you will probably need to use your distribution's package manager to install a C compiler; on Ubuntu, for instance, the relevant package is called `libc-dev`.
 
+## Installation for Windows 10 (Ubuntu on Windows)
+
+This method requires Windows 10 64-bit with the Anniversary edition update (Ubuntu on Windows is not available in 32 bit installations). If you are running Windows 10 but have not yet had the update rolled out to you, you can force the update by visiting https://www.microsoft.com/en-us/software-download/windows10 and selecting the corresponding option.
+
+### Installing Ubuntu on Windows
+
+Go to Windows Settings > Update & Security > For developers. Make sure Developer mode is on (in my instance it required a reboot). Push the start button and type "Turn Windows features on" and select the option in the menu. Check the option for Windows Subsystem for Linux (beta), and again reboot if you are told to do so. Afterward, find "bash" under the Start Menu (I normally just mash the Windows key and type bash). When starting, the system will ask if you want to install Ubuntu from the Windows Store. Do so and wait for it to finish. You will need to set up a username and password. The password will have to be entered several times during this process.
+
+### Setting up folders
+
+One quirk about the Ubuntu on Windows system is that it doesn't seem to like you monkeying around with files inside its root file system. After installation you will see the prompt indicate your current folder is /mnt/c, which is indeed your C: drive. You will probably want to set up a folder inside there to transfer files into and out of Ubuntu. In my case I used
+
+    mkdir Voice
+
+In this folder I could drop files from Windows Explorer, and then later transfer them into Ubuntu's home folder.
+
+Now we will set up a folder in your home directory. This will bring you to your home folder, and I made a folder in here as well
+
+    cd ~
+	mkdir Voice
+	cd Voice
+	
+### Get the HTK
```

Comment:
"Get HTK", not "the HTK". (And similarly below for "compile the HTK")

Also, "an HTK folder", not "a HTK".

Isn't at least some of this redundant, anyways?